frontend/src/extensions/extra-integrations/asciinema/asciinema-frame.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { ClickShield } from '../../../components/markdown-renderer/replace-components/click-shield/click-shield'
+import type { IdProps } from '../../../components/markdown-renderer/replace-components/custom-tag-with-id-component-replacer'
+import React from 'react'
+
+/**
+ * Renders an embedding for https://asciinema.org
+ *
+ * @param id The id from the asciinema url
+ */
+export const AsciinemaFrame: React.FC<IdProps> = ({ id }) => {
+  return (
+    <ClickShield
```

Comment:
Shouldn't we add a key property with the asciinema id here?

Reply:
Not necessary because ids are only valid under one parent.